.eslintrc

```diff
@@ -1,17 +1,16 @@
 {
+    "globals": {
```

Comment:
There is actually a webextension env that you can use (like here: https://github.com/sblask/firefox-simple-form-fill/blob/master/.eslintrc)